src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -1093,7 +1071,7 @@ You can access these logs through the following steps.
 If you need to pass constants or secrets to Appwrite Functions, you can use environment variables. 
 
 {% info title="Appwrite API keys" %}
-If your function is using an Appwrite SDK with an API key, this API key needs to be generated and passed in manually. API keys are not passed by default for security reasons.
+If your function uses an Appwrite SDK with a dynamic API key, go to the function's settings and specify what it can do.
```

Comment:
I would assume the developer needs to select what scopes the API key has access to. If so, it would be good to mention that and link to https://appwrite.io/docs/advanced/platform/api-keys#scopes rather than say

> specify what it can do